indico/modules/events/timetable/client/js/actions.ts

```diff
@@ -149,6 +152,39 @@ export function moveEntry(date: string, entries: TopLevelEntry[]): MoveEntryActi
   return {type: MOVE_ENTRY, date, entries};
 }
 
+export function dnDEntry(entry, eventId, entries: TopLevelEntry[], date: string) {
+  let entryURL: string;
+
+  switch (entry.type) {
+    case 'break':
+      entryURL = breakURL({event_id: eventId, break_id: entry.objId});
+      break;
+    case 'block':
+      entryURL = sessionBlockURL({event_id: eventId, session_block_id: entry.objId});
+      break;
+    case 'contrib':
+      entryURL = contributionURL({event_id: eventId, contrib_id: entry.objId});
+      break;
+    default:
+      entryURL = contributionURL({event_id: eventId, contrib_id: entry.objId});
+  }
+
+  let entryData;
+
+  if (entry.type === 'contrib' || entry.type === undefined) {
+    entryData = {start_dt: moment(entry.startDt).format('YYYY-MM-DDTHH:mm:ss')};
+  } else {
+    const {startDt} = entry;
+    entryData = {start_dt: startDt};
```

Comment:
I'm not a fan of using destructuring assignment there. IMHO it does not increase readability at all compared to this:

```suggestion
    entryData = {start_dt: entry.startDt};
```